tiatoolbox/annotation/storage.py

```diff
@@ -3273,26 +3387,30 @@ def remove(self, key: str) -> None:
         del self._rows[key]
 
     def __getitem__(self, key: str) -> Annotation:
+        """Get an item from the dataset."""
         return self._rows[key]["annotation"]
 
     def __setitem__(self, key: str, annotation: Annotation) -> None:
+        """Implements a method to assign a value to and item."""
```

Comment:
an, not and

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/991322d859c0c22e7c7a1d713362944aa4317198